tests/manage/pv_services/test_delete_project_with_pvc_and_pods.py

```diff
@@ -0,0 +1,165 @@
+"""
+The purpose of this test case is to delete at least 2 projects:
+One project with just pvcs and the other with pvcs attached to a pod
+"""
+import logging
+import random
+import time
+import pytest
+
+from ocs_ci.ocs.exceptions import UnexpectedBehaviour
+from ocs_ci.framework.testlib import ManageTest, tier1
+from ocs_ci.ocs import constants, ocp
+from ocs_ci.ocs.resources import pod
+from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.utility.retry import retry
+from tests import helpers
+
+log = logging.getLogger(__name__)
+
+
+@retry(UnexpectedBehaviour, tries=10, delay=3, backoff=1)
+def check_ceph_used_space():
+    """
+    Check for the used space in cluster
+    """
+    ct_pod = pod.get_ceph_tools_pod()
+    ceph_status = ct_pod.exec_ceph_cmd(ceph_cmd="ceph status")
+    assert ceph_status is not None
+    used = ceph_status['pgmap']['bytes_used']
+    used_in_gb = used / constants.GB
+    global used_space
+    if used_space and used_space == used_in_gb:
+        return used_in_gb
+    used_space = used_in_gb
+    raise UnexpectedBehaviour(
+        f"In Ceph status, used size is keeping varying"
+    )
+
+@tier1
+@pytest.mark.polarion_id("OCS-278")
+class TestDeleteProjectWithPVCAndPods(ManageTest):
+    """
+    Create at least 2 projects, one with just pvcs and the other with pvcs
+    attached to pod, and delete them.
+    """
+    cephfs_pvcs_num = 100
```

Comment:
Read from ENV variables perhaps? We may want in sanity to use 10, on nightly to use 100, on scale to use 1000?

Reply:
Would using pytest.parametrize be a feasible approach?

Reply:
@hadarsharon it will.
You can mark the 100 case with pytest.param(100, marks=pytest.mark.tier1)
and the 1000 one with pytest.param(100, marks=pytest.mark.scale)